ruby_on_rails/rspec.md

```diff
@@ -110,23 +110,10 @@ Commit and push your changes! :tada:
 
 Check that you see `1+2=3` in each app.
 
-## Optional: catch javascript errors
+## Javascript error reporter
 
-If you want to catch Javascript errors in your system tests, you can add the following to the `rails_helper.rb`:
+* Create the module [`spec/support/javascript_error_reporter.rb`](../templates/spec/support/javascript_error_reporter.rb`)
 
-```ruby
-config.after(:each, type: :system, js: true) do
-  errors = page.driver.browser.manage.logs.get(:browser)
-  if errors.present?
-    aggregate_failures 'javascript errors' do
-      errors.each do |error|
-        expect(error.level).not_to eq('SEVERE'), error.message
-        next unless error.level == 'WARNING'
-
-        warn 'WARN: javascript warning'
-        warn error.message
-      end
-    end
-  end
-end
-```
+* Verify that `config.include JavaScriptErrorReporter, type: :system, js: true` is in your [`rails_helper.rb`](../templates/spec/rails_helper.rb)
+
+* Add `js: true` to any system test that should fail when JavaScript errors occur.
```

Comment:
that can be removed since it's implicit IMHO, and might lead to the bad behaviour of adding `js: tre` just to collect javascript errors, with the bad consequence of making the test 10x slower.

Reply:
IMHO (just learned that acronym) it doesn't make a difference. We can't stop anyone from doing that anyways. But I've still removed it. Hope we can finally close this?

Reply:
I don't like if people write "IMHO". It feels a bit like "with all due respect" to me.